models/page_views/optional/snowplow_web_timing_context.sql

```diff
@@ -17,7 +17,7 @@ web_page_context as (
 prep as (
 
     select
-        wp.page_view_id,
+        wp.id,
```

Comment:
hmm, does the `web_page_context` model have a field called `id`? I think at present, `web_page_context` exposes [two fields](https://github.com/fishtown-analytics/snowplow/blob/274c7969d61a552cdc996d00801d7f43cc9b6c03/macros/adapters/default/page_views/snowplow_web_page_context.sql#L34-L35):
- root_id
- page_view_id

Does this work?